src/commands.cpp

```diff
@@ -113,7 +113,16 @@ void CommandGetFA::procresult()
     if (client->json.isnumeric())
     {
         if (it != client->fafcs.end())
-        {
+        {            
+            faf_map::iterator fafsit;
+            for (fafsit = it->second->fafs[0].begin(); fafsit != it->second->fafs[0].end(); )
+            {
+                // move from fresh to pending
+                it->second->fafs[1][fafsit->first] = fafsit->second;
```

Comment:
Without this code, the error callback isn't sent to the app because `FileAttributeFetchChannel::failed()` uses `fafs[1]` only.